docs/usage/attributes.md

```diff
@@ -115,6 +115,14 @@ Cloud providers with supported metadata endpoints and their respective attribute
 * powervs
   - AFTERBURN_POWERVS_INSTANCE_ID
   - AFTERBURN_POWERVS_LOCAL_HOSTNAME
+* scaleway
+  - AFTERBURN_SCALEWAY_HOSTNAME
+  - AFTERBURN_SCALEWAY_INSTANCE_ID
+  - AFTERBURN_SCALEWAY_INSTANCE_TYPE
+  - AFTERBURN_SCALEWAY_PRIVATE_IPV4
+  - AFTERBURN_SCALEWAY_PUBLIC_IPV4
```

Comment:
Nit: Can we change this to be "AFTERBURN_SCALEWAY_IPV4_PUBLIC" Like most of the other providers do? and then do so for the other similar attributes?